### ISSUE TITLE
Is this bundle still maintained?

### ISSUE THREAD

**Issue by dunglas** [lafourchette/SwitchUserStatelessBundle#10]
Hello,

Is this bundle maintained? If it's not the case, @vincentchalamon and I can do it.

ping @jchonde

**Comment by vincentchalamon** [comment]
Up @guillaumevoisin @MLKiiwy @jshedde

**Comment by MLKiiwy** [comment]
I don't know. It's not used in erb-api ... somone know where we use it ?

**Comment by vincentchalamon** [comment]
At least, we use it in CoopTilleuls.

**Comment by vincentchalamon** [comment]
@MLKiiwy As I don't work in LaFourchette anymore, I don't have access to maintain this project. And LaFourchette does not want external members to maintain it

**Comment by MLKiiwy** [comment]
Yeah so maybe you could give you the right on this repo. @jshedde @oversanne it's ok for you ?

**Comment by toofff** [comment]
@jshedde @oversanne up

**Comment by jshedde** [comment]
Sorry, we didn't see the notification. We will get back to you asap.

**Comment by toofff** [comment]
OK @jshedde 

because I like to know when my PR will be merged?

**Comment by jshedde** [comment]
I merged your PR.

**Comment by toofff** [comment]
@jshedde thank's

**Comment by chalasr** [comment]
This repository represents about 300 LoC which are all about avoiding 2 LOC from the symfony/symfony code (a redirection). 
May I propose to implement the feature as a new option for the `switch_user` listener provided by Symfony?

**Comment by dunglas** [comment]
@chalasr, it would be great to have support directly in Symfony.

**Comment by chalasr** [comment]
See https://github.com/symfony/symfony/pull/24260
